### PR TITLE
Bump version and update changelog for beta release 1.22.0-beta.0

### PR DESCRIPTION
**Pull request by ilya-zuyev**
This PR bumps the version and the changelog for beta release 1.22.0-beta.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,76 @@
...
+
+Version Upgrades:
+
+* lib: Upgrade docker go module from 19.03 to 20.10 [#11726](https://github.com/kubernetes/minikube/pull/11726)
+* bump k8s lib to v1.21.2 [#11720](https://github.com/kubernetes/minikube/pull/11720)
+* ISO: Upgrade podman to 3.1.2 [#11704](https://github.com/kubernetes/minikube/pull/11704)
+* Bump google.golang.org/api from 0.47.0 to 0.48.0 [#11646](https://github.com/kubernetes/minikube/pull/11646)
+* Bump github.com/hashicorp/go-getter from 1.5.2 to 1.5.4 [#11719](https://github.com/kubernetes/minikube/pull/11719)
+* Bump github.com/spf13/viper from 1.7.1 to 1.8.0 [#11714](https://github.com/kubernetes/minikube/pull/11714)
+* Upgrade Buildroot to 2021.02 LTS with Linux 4.19 [#11688](https://github.com/kubernetes/minikube/pull/11688)
```
Comment (sharifelgamal) [inline CHANGELOG.md:39]: We don't really care about libs for version upgrades, just keep the podman and buildroot version upgardes and remove the rest.

Comment (ilya-zuyev) [reply CHANGELOG.md:34]: fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,71 @@
 # Release Notes
 
+## Version 1.22.0-beta.0 - 2021-06-28
+
+Features:
+
+* Add auto pause on arm64 image [#11743](https://github.com/kubernetes/minikube/pull/11743)
+* Addon list: Add info on each addon's maintainer  [#11753](https://github.com/kubernetes/minikube/pull/11753)
```
Comment (medyagh) [inline CHANGELOG.md:8]: change this with back ticks with lower case
`addon list`: ...

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,71 @@
 # Release Notes
 
+## Version 1.22.0-beta.0 - 2021-06-28
+
+Features:
+
+* Add auto pause on arm64 image [#11743](https://github.com/kubernetes/minikube/pull/11743)
```
Comment (medyagh) [inline CHANGELOG.md:7]: `auto-pause` addon: add support for arm64

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,71 @@
...
+* Addon list: Add info on each addon's maintainer  [#11753](https://github.com/kubernetes/minikube/pull/11753)
+* Add ability to pass 'max' value to memory and cpus flags [#11692](https://github.com/kubernetes/minikube/pull/11692)
```
Comment (medyagh) [inline CHANGELOG.md:9]: add ability to pass `max` to `--cpu` and `--memory` flags

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,71 @@
...
+
+Bugs:
+
+* Fix --base-image caching for images specified by name:tag [#11603](https://github.com/kubernetes/minikube/pull/11603)
+* Fix embed-certs global config [#11576](https://github.com/kubernetes/minikube/pull/11576)
+* Fix a download link to use arm64 instead of amd64 [#11653](https://github.com/kubernetes/minikube/pull/11653)
+* Move daemon cache check before file cache check [#11690](https://github.com/kubernetes/minikube/pull/11690)
```
Comment (medyagh) [inline CHANGELOG.md:16]: reword what does it actually fix?  how about this
`fix downloading duplicate base image`

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,71 @@
...
+* Move node config deletion out of drainNode and into Delete [#11731](https://github.com/kubernetes/minikube/pull/11731)
```
Comment (medyagh) [inline CHANGELOG.md:17]: change wording to what it actually does.
`fix multi-node loosing track of nodes after second restart`

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,71 @@
...
+* gcp-auth: do not override existing environment variables in pods [#11665](https://github.com/kubernetes/minikube/pull/11665)
+
+Minor improvements:
+
+* Allow running amd64 binary on M1 [#11674](https://github.com/kubernetes/minikube/pull/11674)
+* Upgrade containerd config [#11632](https://github.com/kubernetes/minikube/pull/11632)
```
Comment (medyagh) [inline CHANGELOG.md:23]: improve containerd experience on cgroup v2

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,71 @@
...
+* Improve French locale [#11728](https://github.com/kubernetes/minikube/pull/11728)
+* Do not return an error from Systemd.ForceStop(svc) if svc is already stopped [#11667](https://github.com/kubernetes/minikube/pull/11667)
```
Comment (medyagh) [inline CHANGELOG.md:25]: what does it do for user, change wording how does this fix user's problem 
something like `Fix UI error on contained error spam...`

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,71 @@
...
+* Let windows users use the LC_ALL env var to set locale [#11721](https://github.com/kubernetes/minikube/pull/11721)
```
Comment (medyagh) [inline CHANGELOG.md:26]: international languages: allows using LC_ALL env to set local local language for windows

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,71 @@
...
+* Remove unused config options [#11668](https://github.com/kubernetes/minikube/pull/11668)
+* Fix intersected log boxes when running with --alsologtostderr [#11694](https://github.com/kubernetes/minikube/pull/11694)
```
Comment (medyagh) [inline CHANGELOG.md:28]: not related to user, remove

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,71 @@
...
+* gcp-auth: do not override existing environment variables in pods [#11665](https://github.com/kubernetes/minikube/pull/11665)
+
+Minor improvements:
+
+* Allow running amd64 binary on M1 [#11674](https://github.com/kubernetes/minikube/pull/11674)
+* Upgrade containerd config [#11632](https://github.com/kubernetes/minikube/pull/11632)
+* Improve French locale [#11728](https://github.com/kubernetes/minikube/pull/11728)
+* Do not return an error from Systemd.ForceStop(svc) if svc is already stopped [#11667](https://github.com/kubernetes/minikube/pull/11667)
+* Let windows users use the LC_ALL env var to set locale [#11721](https://github.com/kubernetes/minikube/pull/11721)
+* Remove unused config options [#11668](https://github.com/kubernetes/minikube/pull/11668)
```
Comment (medyagh) [inline CHANGELOG.md:27]: this is code clean up not related to user, remove

**Review by medyagh — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 10 above)
Comment (ilya-zuyev) [reply CHANGELOG.md:28]: it was visible to uses and affected UX

File: CHANGELOG.md (hunk 2 above)
Comment (ilya-zuyev) [reply CHANGELOG.md:8]: where should I put back ticks?

File: CHANGELOG.md (hunk 11 above)
Comment (ilya-zuyev) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 9 above)
Comment (ilya-zuyev) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 8 above)
Comment (ilya-zuyev) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 7 above)
Comment (ilya-zuyev) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 6 above)
Comment (ilya-zuyev) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 5 above)
Comment (ilya-zuyev) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 4 above)
Comment (ilya-zuyev) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 3 above)
Comment (ilya-zuyev) [reply CHANGELOG.md]: done

**Review by medyagh — APPROVED** [review]

**Comment by ilya-zuyev** [comment]
https://github.com/kubernetes/minikube/pull/11812 - integration tests passed for that PR, looks like we're ok to merge this one

**Pull request merged**